src/_pytest/pytester.py

```diff
@@ -357,16 +357,34 @@ def clear(self) -> None:
 
 @pytest.fixture
 def linecomp() -> "LineComp":
+    """
+    Return a :class" `LineComp`instance usable to check that the input linearly contains a sequence of strings.
+
+    see: https://docs.pytest.org/en/latest/_modules/_pytest/pytester.html
```

Comment:
I can see why you added this link, but linking to source code seems a bit inappropriate to me. I'd omit it for now (also in the others).